spec/features/user_can_edit_an_artists_attributes_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../rails_helper'
+
+describe "When a user is on the show page of an artist" do
+  scenario "they can click edit and fill out a form and change an artists attributes" do
+    marley_path = "http://cps-static.rovicorp.com/3/JPG_400/MI0003/146/MI0003146038.jpg"
+    sinbad_path = "https://pbs.twimg.com/profile_images/477764918326743040/q8_9xrhm.png"
+    artist = Artist.create(name: "Sinbad", image_path: sinbad_path)
+
+    visit(artist_path(artist))
+
+    click_on("Edit")
+
+    expect page.has_current_path?(edit_artist_path(artist))
```

Comment:
@CjMoore I know you mentioned you thought these looked sparse, but I might actually skip the visit to the `artist_path`. I think you could drop right into the `edit_artist_path` to test the Edit. Don't think there's as much value in testing that the link to the edit view works.